android/src/main/java/com/oney/WebRTCModule/CameraCaptureController.java

```diff
@@ -86,7 +91,7 @@ protected VideoCapturer createVideoCapturer() {
      * Helper function which tries to switch cameras until the desired facing mode is found.
      *
      * @param desiredFrontFacing - The desired front facing value.
-     * @param tries - How many times to try switching.
+     * @param tries              - How many times to try switching.
```

Comment:
Please avoid making any unrelated changes. It complicates the diff for no good reason. I'm working on integrating clang-format to take care of this automatically.